.github/ISSUE_TEMPLATE/new-bridge-method.md

```diff
@@ -4,25 +4,46 @@ about: Suggest a new bridge method
 title: ''
 labels: ''
 assignees: ''
-
 ---
 
 **Message type**: `MESSAGE_TYPE`
 
+**Originator**: (specifies who initiates the request: Web or Native)
+
 ## Description
+Provide a description of the method and the behaviour of the apps
 
 ## Differences between iOS/Android (if any)
+To be filled only if apply
 
-## Request/Response shape
+## Method
+**Signature**
+```ts
+methodName: ({ param1: string, param2: string }) => Promise<{responseField: string}> 
+```
```

Comment:
I wouldn't include this in the issue. The issue should be a spec of the contract between native and web (the mesage type and payload shape). The method name is just an implementation detail of the web side.